### DISCUSSION TITLE
Scheduled Jobs Page

### DISCUSSION THREAD

**Discussion by lthoa** [oqtane/oqtane.framework#1657] [Q&A] [answered]
Are we missing the "Add new Job" button or you don't want to add new job record?
I have the EmailJob module and want to add it here.

![image](https://user-images.githubusercontent.com/6874322/133018079-c7923fd8-5a08-4986-a8c3-2800d5879fc3.png)

**Comment by leigh-pointer** [comment, marked as answer]
Schedule Jobs are added dynamically through an implemented interface!

Check out the notifications job in the oqtane.server project.
________________________________
From: lthoa ***@***.***>
Sent: Monday, September 13, 2021 5:12:33 AM
To: oqtane/oqtane.framework ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [oqtane/oqtane.framework] Scheduled Jobs Page (Discussion #1657)


Are we missing the "Add new Job" button or you don't want to add new job record?
I have the EmailJob module and want to add it here.

[image]<https://user-images.githubusercontent.com/6874322/133018079-c7923fd8-5a08-4986-a8c3-2800d5879fc3.png>

**Comment by lthoa** [comment]
Thank @leigh-pointer
